chain/rpc/src/indexer.rs

```diff
@@ -96,7 +96,7 @@ impl<P: JsonRpcClient + 'static> HoprIndexerRpcOperations for RpcOperations<P> {
 
                         // Range of blocks to fetch is always bounded
                         let range_size = self.cfg.max_block_range_fetch_size.min(latest_block - from_block);
-                        debug!("polling logs from blocks #{from_block} - #{latest_block} (range size {range_size})");
+                        info!("polling logs from blocks #{from_block} - #{latest_block} (range size {range_size})");
```

Comment:
I'd avoid making this an info. There are higher level logic outputs for block processing.